config/config.go

```diff
@@ -0,0 +1,76 @@
+package config
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/knadh/koanf/parsers/json"
+	"github.com/knadh/koanf/providers/file"
+	"github.com/knadh/koanf/providers/structs"
+	"github.com/knadh/koanf/v2"
+)
+
+var ENV_FILE_NAME = "env.json"
```

Comment:
This is signaling itself as a constant by using all caps, but in Go, the convention is not to use caps for a constnat.

If this will never be reassigned, let's make it a constant and an idiomatic variable name.

Reply:
I learned something today :)